www/sass/_mixin.scss

```diff
@@ -0,0 +1,3 @@
+/*************************/
```

Comment:
IMO this file should be named `_mixins.scss`. Same with `_vars.scss`. Hey everyone what do you think ? @UnderDevCrew/dev ?